HaulageApplication/Haulage/App.xaml.cs

```diff
@@ -8,4 +8,13 @@ public App()
 
 		MainPage = new AppShell();
 	}
+
+    protected override Window CreateWindow(IActivationState activationState)
+    {
+        Window window = base.CreateWindow(activationState);
```

Comment:
what is this doing?

Reply:
removed - was trying to get table refresh working but no joy